authors/apps/authentication/models.py

```diff
@@ -76,6 +76,11 @@ class User(AbstractBaseUser, PermissionsMixin):
     # falsed.
     is_staff = models.BooleanField(default=False)
 
+    # The `is_verified` flag is expected by our platform to determine whether
+    # a user has verified their email address.Users can only login if this is
+    # true
+    is_verified = models.BooleanField(default=False)
```

Comment:
@ssewilliam, I think you will not need this line so far. @lytes20 has a line of code doing something similar to what the line in the subject does. You could link up with him and work out something.

Reply:
@ssewilliam  Do u need this in the current PR